source/_posts/front-commerce-2.2.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Release: Front-Commerce 2.2"
+date: 2020-11-18
+---
+
+Front-Commerce 2.2 has a heavy focus on Magento 1 features thanks to new clients migrating their Legacy Magento 1 shops on Front-Commerce. Read more about this version.
+
+<!-- more -->
+
+## Guest Checkout
+
+It is now possible to complete a Checkout in Front-Commerce without being logged. This is a frequently asked feature as it can really boost your sales. This also means that we've made all the changes necessary to the theme and that it will now be possible to add the Guest Checkout to Magento 2 with very few changes. Stay tuned!
+
+## More product types
+
+In Front-Commerce we already supported simple products and configurable products which are the most commonly used types in the Magento ecosystem. In Front-Commerce's Magento 1 integration it is now possible to use Bundle Products and Virtual Products.
+
+Virtual products are especially exciting because this means that it is now possible to remove completly the shipping step from the checkout if a customer only buys virtual items.
+
+Just like the Guest Checkout this also means that we're close to add the support for these types in Magento 2's integration.
+
+For Magento 2, we've added support for Grouped Products.
+
+## Multiple currencies
+
+In Magento there are multiple ways to sell your products in different currencies. You can either configure a different website or set multiple currencies on a single store. Each one has its pros and cons, but luckily for you, we now support both ways in Magento 1.
+
+Please refer to [Multiple currencies](/docs/advanced/production-ready/multistore.html#Multiple-currencies) for more details.
+
```

Comment:
I'd add a section about our Wordpress module that reached 1.0.0 too.

Here is a draft:

## Wordpress module reach 1.0

Since a few years, Front-Commerce allows to consume CMS content, blog posts and other ACF data types from Wordpress. As part of this Front-Commerce release, we took some time to clean things up and update the Wordpress module to leverage new features added in Front-Commerce.

The Wordpress module now has an even wider coverage of native features such as archives, categories, permalinks configuration and RSS feeds. We've improved default SEO metadata and used Front-Commerce `<Wysiwyg>` and `<Image>` components to bring performance and extensibility to Wordpress content too!

Don't want to choose? It is now possible to mix CMS pages from different sources, so you could migrate progressively your content from Magento to Wordpress.